pallets/parachain-staking/src/tests.rs

```diff
@@ -93,16 +93,16 @@ fn set_total_selected_fails_if_above_blocks_per_round() {
 }
 
 #[test]
-fn set_total_selected_passes_if_equal_to_blocks_per_round() {
+fn set_total_selected_fails_if_equal_to_blocks_per_round() {
```

Comment:
can you add a test `set_total_selected_passes_if_greater_than_blocks_per_round` ?